src/crawlee/_request.py

```diff
@@ -127,15 +127,14 @@ class BaseRequestData(BaseModel):
     method: HttpMethod = 'GET'
     """HTTP request method."""
 
-    headers: Annotated[HttpHeaders, Field(default_factory=HttpHeaders())] = HttpHeaders()
```

Comment:
I believe `Field(default_factory=HttpHeaders())` is actually used.

The `= HttpHeaders()` is just for the Mypy. Otherwise, it approaches it as a mandatory argument in the constructor.